tools/sdk/tribeca/configurations.ts

```diff
@@ -0,0 +1,48 @@
+import { PublicKey } from '@solana/web3.js'
+import ATribecaConfiguration from './ATribecaConfiguration'
+
+class SaberTribecaConfiguration extends ATribecaConfiguration {
+  public readonly locker = new PublicKey(
+    '8erad8kmNrLJDJPe9UkmTHomrMV3EW48sjGeECyVjbYX'
+  )
+
+  public readonly token = {
+    name: 'SBR - Saber Protocol Token',
+    mint: new PublicKey('Saber2gLauYim4Mvftnrasomsv6NvAuncvMEZwcLpD1'),
+    decimals: 6,
+  }
```

Comment:
feels like there is something to do with the token data listed in `utils/splTokens.ts`

Reply:
right we could use the info from `SPL_TOKENS` directly :)